kubernetes/omsagent.yaml

```diff
@@ -746,14 +746,23 @@ spec:
       port: 25227
       targetPort: in-rs-tcp
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can you please add comment about how to handle this version across k8s versions ?. This will be helpful when we make changes to AKS yaml and when use this yaml for local development and testing.